manifests/t/the-sz/Banks/1.02/the-sz.Banks.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: the-sz.Banks
+PackageVersion: 1.02
+PackageLocale: en-US
+Publisher: the sz development
+PublisherUrl: https://the-sz.com/
+PrivacyUrl: https://the-sz.com/products/privacy.php
+Author: the sz development
+PackageName: Banks
+PackageUrl: https://the-sz.com/products/banks/
+License: Copyright (c) the-sz.com

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate